### ISSUE TITLE
SameDiff/libnd4j: deconv op output size (or padding) configuration

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#8347]
We already have this for TF import already, i.e., deconv2d_tf (and soon deconv3d_tf) that support an output size argument.

But compare our deconv2d op to TF's conv2d_transpose: note the addition of the output_shape argument.
https://www.tensorflow.org/api_docs/python/tf/nn/conv2d_transpose

This should be optional. Note it's functionally equivalent to Keras' output padding:
https://keras.io/layers/convolutional/ -> Conv2DTranspose -> output_padding

See also Pg26: https://arxiv.org/pdf/1603.07285.pdf
"The constraint on the size of the input i can be relaxed by introducing another parameter a ∈ {0, ..., s−1} that allows to distinguish between the s different cases that all lead to the same i'"

**Comment by agibsonccc** [comment]
Closing — SameDiff has been completely rewritten with a new DSP (Dynamic Shape Plan) execution engine, native plan compilation, CUDA graph replay, and Triton compilation. The architecture referenced in this issue no longer exists. If you still experience this issue with the upcoming release, please open a new issue.
